### PR TITLE
Adding a template for change plan requests/proposals.

### PR DESCRIPTION
**Pull request by TopherGopher**

### CODE REVIEW

**Comment by cyberswat** [comment]
This is in response to https://github.com/drud/aws-infra/issues/8 ... The purpose of the ticket is to provide a community based template that we can refer people to when they would like to initiate change.

File: development/proposal_template.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Project Management and Costs
+Briefly outline the proposed size and skills of the team, plus the timing, costs, and co-commitments. Provide a concise appraisal of the resources, the timing and any co-commitments needed for successful implementation of the project. Graphs are very useful in this section.
+
+## Scope of Work
+This section is the heart of your proposal and should be accessible to the layman. Avoid technical terminology and use diagrams, charts and tables to express complex points clearly. Expand on the points from the executive summary.
+### The Current Situation
+Deepdive the points made in the current situation above. Provide additional context and describe the problem more completely.
+
+
+### The Ideal Situation
```
Comment (cyberswat) [inline development/proposal_template.md]: This should probably not be repeated. Same with The Proposed Solution

File: development/proposal_template.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Describe the 'why' 'what' and 'how'. Why is this important? What will the outcome be? 
+
+### The Proposed Solution
+This is your approach in detail, still free of technical terms and accessible still to the layman. This is where you introduce your solution and describe why your approach is the best available. Make sure to note any relevant costs, benefits and risks. The aim is to describe the value you will deliver using your approach.
+
+### Managing the Project
```
Comment (cyberswat) [inline development/proposal_template.md]: We don't want people considering people and budget.  We want them to consider timelines of major pieces, but it's probably too much to ask for more.

**Review by cyberswat — CHANGES_REQUESTED** [review]

**Comment by TopherGopher** [comment]
I have combined a few sections and removed references to things such as cost and team members. See if you like what I did.

File: development/proposal_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+## The Current Situation
```
Comment (cyberswat) [inline development/proposal_template.md:1]: This looks like it was copied out of a pdf.  It needs to be written from our perspective using our words.

**Review by cyberswat — CHANGES_REQUESTED** [review]

**Comment by cyberswat** [comment]
This should be resolved by #4

**Pull request closed without merging**